pkgs/by-name/tk/tkdiff/package.nix

```diff
@@ -1,17 +1,27 @@
 { diffutils, fetchzip, lib, makeBinaryWrapper, stdenv, tk }:
 
-stdenv.mkDerivation (finalAttrs: {
```

Comment:
This change is unrelated, please revert. Also there isn't really a good reason to remove this.